src/components/SwitchCheckboxIcon/SwitchCheckboxIcon.scss

```diff
@@ -1,67 +1,314 @@
+            height:16px;
+            z-index: 99;
+            margin:0px;
+            position: relative;
+        }
+        .trueRadio { 
+            input:checked + .switchRadio {
+                left:1px;
+                opacity: 1;
+                visibility: visible;
+            }
+        }
+        .nullRadio { 
+            input:checked + .switchRadio {
+                left:50%;
+                opacity: 1;
+                visibility: visible;
+                transform: translateX(-50%);
+            }
+         }
+        .falseRadio { 
+            input:checked + .switchRadio {
+                right:1px;
+                opacity: 1;
+                visibility: visible;
+            }
+        }
+        .Accept,
+        .Cancel {
             position: absolute;
-            z-index: 2;
-            &:checked + .InnerSpan {
-                background-color: color(blue, base);
-            .innerCircle {
-                right:2px;
-                left: auto;
-                transition: all .3s ease-in-out;
-                -moz-transition: all .3s ease-in-out;
-                -webkit-transition: all .3s ease-in-out;
-                background: color(white);
-                .Cancel {
-                    display: none;
+            top: 1.5px;
+            display: none;
+        }
+        .Accept {
+            left: -3px;
+        }
+        .Cancel {
+            right: 1.5px;
+        }
+    }
+}
+.switchNormal {
+    .outeWrap {
+        background:color(blue);
+        border-color:color(blue);
+        width: 4.2rem;
+        .Accept,
+        .Cancel {
+            display: none!important;
+        }
+        .trueRadio {
+            
+            input {
+                &:checked + .switchRadio {
+                    left:-6px;
+                    opacity: 1;
+                    visibility: visible;
+                }
+            }
+        }
+        .nullRadio {
+            input {
+                &:checked + .switchRadio {
+                    left:50%;
+                    opacity: 1;
+                    visibility: visible;
+                    transform: translateX(-50%);
+                }
+            }
+        }
+        .falseRadio {
+            
+            input {
+                &:checked + .switchRadio {
+                    right:-6px;
+                    opacity: 1;
+                    visibility: visible;
+                }
+            }
+        }
+    }  
+}
+.switchTrueFalse {
+    .outeWrap {
+        background:color(blue);
+        border-color:color(blue);
+        width: 5.4rem;
+        .trueRadio {
+          input {
+                &:checked + .switchRadio {
+                    left:-2px;
+                    opacity: 1;
+                    visibility: visible;
+                    &:after {
+                        content:'TRUE';
+                        font-size: 1rem;
+                        color:color(white);
+                        left: -29px;
```

Comment:
I see you're mixing relative (line 126 width: 5.4rem) and absolute (left: -29px) dimensions. This will create issues if we scale the base unit size. Please use all relative units. I think our base unit is 10px, so it's very simple to use -2.9rem in place of -29px.